src/puppetlabs/ezbake/core.clj

```diff
@@ -568,6 +569,8 @@ Additional uberjar dependencies:
                                                       "puppetlabs.trapperkeeper.main")
      :java-args                          (get-local-ezbake-var lein-project :java-args
                                                       "-Xmx192m")
+     :tk-args                            (get-local-ezbake-var lein-project :tk-args
+                                                      "-test")
```

Comment:
What's this `"-test"` bit for?

Reply:
whoops forgot to remove that, it was for testing